tests/ecdhp256-test.c

```diff
@@ -89,8 +93,134 @@ int main()
 	ok = ok && compare_and_print(64, point_compressed, compressed7);
 
 
+	printf("%s\n", "Wycheproof tests ECDH: ");
+		
+	for (int i = 0 ; i< sizeof(w_vectors)/sizeof(ecdhp256_w_i); i++)
+	{
+		printf("ECDH Wycheproof Test %d ", i );
+
+		uint8_t* decompressedPoint = (uint8_t*) malloc (sizeof (uint8_t) * 64);
+		bool flagDecompress = Hacl_P256_decompression_not_compressed_form(w_vectors[i].publicKey, decompressedPoint);
```

Comment:
the documentation states that Hacl_P256_decompression_not_compressed_form returns true if and only if the decompression was successful, but the name `flagDecompress` seems to indicate something else -- let's use `ok`, or `success`, or something else